iNaturalist.xcodeproj/project.pbxproj

```diff
@@ -1358,47 +1358,48 @@
 			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
 			showEnvVarsInLog = 0;
 		};
-		9D5253E631060D844242E2BF /* Copy Pods Resources */ = {
+		89B4C2BF1B665A74005B948A /* Run Script */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
-			name = "Copy Pods Resources";
+			name = "Run Script";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-iNaturalistTests/Pods-iNaturalistTests-resources.sh\"\n";
-			showEnvVarsInLog = 0;
+			shellScript = "\"${PODS_ROOT}/Fabric/Fabric.framework/run\" 345e342e4072648b2f39f3eb6ba91e02002087e3 772e463271ce4ae8d80b4c3f53bc00109ac88f7d6b23c21faeb89dff06cc74a6";
```

Comment:
Your change here will require all forks to have Fabric & Crashlytics installed in order to build, which I would prefer not to have. Please back out your changes to the project.pbxproj unless they're relevant to all forks of iNaturalist, including the main iNaturalist App Store app.